common/lib/common-definitions/tsconfig.json

```diff
@@ -1,12 +1,9 @@
 {
-	"extends": [
-		"../../build/build-common/tsconfig.base.json",
-		"../../build/build-common/tsconfig.cjs.json",
-	],
+	"extends": "../../../common/build/build-common/tsconfig.node16.json",
 	"include": ["src/**/*"],
 	"exclude": ["dist", "node_modules"],
 	"compilerOptions": {
 		"rootDir": "./src",
-		"outDir": "./dist",
+		"outDir": "./lib",
 	},
```

Comment:
With the intention of being type-only this should use `"emitDeclarationOnly": true,` to only emit .d.ts files.